Doc/library/datetime.rst

```diff
@@ -280,20 +280,26 @@ Class attributes:
    The smallest possible difference between non-equal :class:`timedelta` objects,
    ``timedelta(microseconds=1)``.
 
-Note that, because of normalization, ``timedelta.max`` > ``-timedelta.min``.
+Note that, because of normalization, ``timedelta.max > -timedelta.min`` is true.
```

Comment:
This might be clearer as:
```suggestion
Note that, because of normalization, ``timedelta.max`` is greater than ``-timedelta.min``.
```
But I think the existing phrasing is fine too.